### PR TITLE
Fix un-init member warnings in DataFormats/SiStripCluster

### PR DESCRIPTION
**Pull request by mrodozov**
Put init value = 0 for the firstStrip_ member to stop gcc7 complain https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc6_amd64_gcc700/CMSSW_9_4_X_2017-09-27-1100/DataFormats/TrackerRecHit2D

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by mrodozov** [comment]
please test

File: DataFormats/SiStripCluster/interface/SiStripCluster.h [hunk 1]
```diff
@@ -15,28 +15,28 @@ class SiStripCluster  {
...
 
   explicit SiStripCluster(const SiStripDigiRange& range);
 
   template<typename Iter>
   SiStripCluster(const uint16_t& firstStrip, 
 		 Iter begin, Iter end ):
-	 amplitudes_(begin,end), firstStrip_(firstStrip), 
+	 amplitudes_(begin,end), firstStrip_(firstStrip) {}
   // ggiurgiu@fnal.gov, 01/05/12
   // Initialize the split cluster errors to un-physical values.
```
Comment (perrotta) [inline DataFormats/SiStripCluster/interface/SiStripCluster.h]: This comment should be moved close to L95, where there is now the actual initialization of the cluster errors to un-physical values

**Comment by perrotta** [comment]
ping @mrodozov : this PR only misses moving the comment to the place where the line it refers to has been moved

File: DataFormats/SiStripCluster/interface/SiStripCluster.h (hunk 1 above)
Comment (mrodozov) [reply DataFormats/SiStripCluster/interface/SiStripCluster.h]: @perrotta, PR is updated

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical: fix comp warnings in gcc7 by initializing private parameters for all SiStripCluste(...) constructors
- Jenkins shows no effects (none expected)

**Comment by davidlange6** [comment]
+1

**Pull request merged**